samples/bluetooth/central_past/src/main.c

```diff
@@ -293,7 +293,7 @@ int main(void)
 		sync_create_param.options = 0;
 		sync_create_param.sid = per_sid;
 		sync_create_param.skip = 0;
-		sync_create_param.timeout = 0xaa;
+		sync_create_param.timeout = 1000; /* 10 seconds */
```

Comment:
What we have done in the LE audio samples/tests is that we use the periodic advertising interval to calculate a reasonable timeout. 

That way it will always work with any advertiser regardless of their setup; this change here will still fail for advertisers using an interval of e.g. 12 seconds. 

We've seen several platforms at UPF using a hardcoded timeout like here, and not only is it hard to debug, but it also doesn't really make sense to hardcode it (for example if the interval is 20ms, there's no point in waiting 10 seconds before a timeout)

Reply:
I agree. That sounds like a better approach. I will update this PR accordingly